db/migrate/20150902080000_create_people.rb

```diff
@@ -0,0 +1,23 @@
+class CreatePeople < ActiveRecord::Migration
+  def change
+    create_table :people, id: :uuid do |t|
+      t.string :nickname
+      t.string :fullname, null: false
+      t.string :position
+      t.string :image
+      t.string :github
+      t.string :blog
+      t.string :twitter
+      t.string :facebook
+      t.string :website
+      t.string :email
+      t.string :skills, default: ''
+      t.string :technologies, default: ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.